install-moveit2/source/index.markdown

```diff
@@ -77,7 +79,7 @@ Download the repository and install any dependencies:
 
     git clone https://github.com/ros-planning/moveit2.git
     vcs import < moveit2/moveit2.repos
```

Comment:
We still uıse moveit2_rolling.repos and  moveit2_galactic.repos until a galactic and rolling sync that includes our latest releases.

Reply:
all the required packages are in moveit2.repos, I think we can remove the others already

Reply:
Rolling passes without the repos with the latest sync. Galactic fails because there are no syncs since release yet :(.

Reply:
See: https://github.com/vatanaksoytezer/moveit2/runs/2964191063?check_suite_focus=true

Reply:
... ok I think it's all there now